<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFPlotterCopyToTableDialog</class>
 <widget class="QDialog" name="QFPlotterCopyToTableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add plot data to table ...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="labDescription">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>file:</string>
       </property>
       <property name="buddy">
        <cstring>cmbRDR</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QFRDRTableComboBox" name="cmbRDR"/>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="chkNewTable">
       <property name="text">
        <string>create new:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edtNewTableName">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="chkOnlyForPlots">
       <property name="toolTip">
        <string>if activated this tool will only copy the data really used for the selected plot. If unchecked additional column might be copied!</string>
       </property>
       <property name="text">
        <string>copy only data that is required for plots</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="edtPrefix">
       <property name="toolTip">
        <string>this text is prepended to every column name in the table (e.g. use this to identify where which column came from!)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>column name prefix:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string> Add to graph? </string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radAddDataOnly">
        <property name="text">
         <string>only add data to table</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radAlsoAddGraphs">
        <property name="text">
         <string>also add graphs</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widGraph" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="3">
          <widget class="QFRDRColumnGraphsComboBox" name="cmbGraph"/>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="edtNewGraphName">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>graph to add to:</string>
           </property>
           <property name="buddy">
            <cstring>cmbGraph</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="chCreateNewGraph">
           <property name="text">
            <string>create new</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkShowTableEditor">
     <property name="text">
      <string>show table editor after insert</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFRDRTableComboBox</class>
   <extends>QComboBox</extends>
   <header>qfrdrtablecombobox.h</header>
   <slots>
    <signal>currentColumnGraphicsChanged(QFRDRColumnGraphsInterface*)</signal>
    <signal>currentTableChanged(QFRDRTableInterface*)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>QFRDRColumnGraphsComboBox</class>
   <extends>QComboBox</extends>
   <header>qfrdrtablecombobox.h</header>
   <slots>
    <slot>setColumnGraphs(QFRDRColumnGraphsInterface*)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QFPlotterCopyToTableDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QFPlotterCopyToTableDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radAlsoAddGraphs</sender>
   <signal>toggled(bool)</signal>
   <receiver>widGraph</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbRDR</sender>
   <signal>currentColumnGraphicsChanged(QFRDRColumnGraphsInterface*)</signal>
   <receiver>cmbGraph</receiver>
   <slot>setColumnGraphs(QFRDRColumnGraphsInterface*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
